src/slider/SliderTrack.ts

```diff
@@ -4,35 +4,37 @@
  * We improved the hook [useSlider](https://github.com/chakra-ui/chakra-ui/blob/af613020125265914a9dcb74c92a07a16aa4ff8e/packages/slider/src/use-slider.ts)
  * to work with Reakit System
  */
-import { createComponent, createHook } from "reakit-system";
-import { BoxHTMLProps, useBox } from "reakit";
 import { useId } from "@chakra-ui/hooks";
+import { BoxHTMLProps, useBox } from "reakit";
 import { mergeRefs, dataAttr } from "@chakra-ui/utils";
+import { createComponent, createHook } from "reakit-system";
 
-import { UseSliderReturn } from "./SliderState";
 import { SLIDER_TRACK_KEYS } from "./__keys";
+import { UseSliderReturn } from "./SliderState";
 
 export type useSliderTrackOptions = UseSliderReturn & {
   /**
-   * The base `id` to use for the sliderTrack and it's components
+   * The base `id` to use for the sliderTrack
    */
   id?: string;
 };
 
 export const useSliderTrack = createHook<useSliderTrackOptions, BoxHTMLProps>({
-  name: "useSliderTrack",
-  compose: [useBox],
+  name: "SliderTrack",
+  compose: useBox,
   keys: SLIDER_TRACK_KEYS,
 
   useProps(options, { ref: htmlRef, style: htmlStyle, ...htmlProps }) {
+    const { refs, state, styles } = options;
+
     const id = useId(options.id, "slider-track");
 
     return {
-      id,
-      ref: mergeRefs(htmlRef, options.refs.trackRef),
-      "data-disabled": dataAttr(options.state.isDisabled),
-      style: { ...options.styles.trackStyle, ...htmlStyle },
       ...htmlProps,
+      id,
+      "data-disabled": dataAttr(state.isDisabled),
+      style: { ...htmlStyle, ...styles.trackStyle },
+      ref: mergeRefs(htmlRef, refs.trackRef),
```

Comment:
We can maybe use Reakit's useForkRef too right? i saw the code it's very similar just that merge refs can take multiple arguments and useForkRef takes just two. 

*(it won't matter but jut to reduce duplicate utils & save few extra bytes haha)*

Reply:
Yup, I can totally do it. No issues. Just a matter of preference.

I thought since we are using it from Chakra, so did not mean to change it.

Reply:
Okay! although if you change it make sure it works as expected. :D

Reply:
Yup, it works as it is, both has the same logic behind.

I have experimented with both of them.